borrowing_service/models.py

```diff
@@ -6,12 +8,13 @@
 
 
 class Borrowing(models.Model):
-    borrow_date = models.DateField(auto_now_add=True,)
+    borrow_date = models.DateField()

```

Comment:
Why did you remove `auto_now_add` parameter? We need to have this field as autofilled

Reply:
Cause i need `borrow_date` in validation `borrow_date > expected_borrow_date`, so i add borrow_date during creation